images/scripts/rhel.setup

```diff
@@ -329,9 +329,6 @@ fi
 # and more convenient interactive debugging
 dnf --setopt=install_weak_deps=False install -y cockpit
 
-# For debugging udisks/storaged crashes
-debuginfo-install -y udisks2
```

Comment:
Please only drop it on centos, it seems to work well on RHEL.

Reply:
ack, "Let's kill that debuginfo-install." sounded too strinct to keep it on rhel 😄